.travis.yml

```diff
@@ -18,7 +18,7 @@ before_install:
     - conda update --all --yes
     - travis_retry conda create --yes -n TEST python=$TRAVIS_PYTHON_VERSION --file requirements.txt
     - source activate TEST
-    - travis_retry conda install --yes --file requirements-dev.txt
+    - travis_retry pip install -r requirements-dev.txt
```

Comment:
OK, makes sense -- I still see pip and conda get tangled up in dependencies, but for running tests on a CI, a conda install followed by a pip install is fine.

BTW, maybe a conda_requirements.txt and a requirements.txt (for pip)? they are mostly the same, but not always (sigh.)